README.md

```diff
@@ -1,223 +1,350 @@
-# STRATUM
+# Stratum V2 (Sv2) Proxy
+
```

Comment:
Perhaps after the title, we can have here a paragraph about what Stratum is, just to give a good context?